config/locales/en.yml

```diff
@@ -34,6 +34,9 @@ en:
 
   errors:
     format: "%{message}"
+    messages:
+      content_type_invalid: "has an invalid content type: %{content_type}"
+      file_size_out_of_range: "size %{file_size} is not between required range"
```

Comment:
Should this be simplified to "file size is too large", since we aren't specifying a minimum?

Reply:
chnaged to "file size %{file_size} is too large".